mage_ai/data_preparation/models/block/dbt/dbt_adapter.py

```diff
@@ -5,17 +5,17 @@
 
 import dbt.flags as flags
 import pandas as pd
-from dbt.adapters.base import BaseRelation, Credentials
+from dbt.adapters.base import BaseRelation
 from dbt.adapters.factory import (
     Adapter,
     cleanup_connections,
     get_adapter,
     register_adapter,
     reset_adapters,
 )
-from dbt.config.profile import read_user_config
 from dbt.config.runtime import RuntimeConfig
-from dbt.contracts.connection import AdapterResponse
+from dbt.config.utils import read_user_config
+from dbt.contracts.connection import AdapterResponse, Credentials
```

Comment:
- `from dbt.config.project import read_project_flags` -> It seems this is substituting the deprecated `read_user_config`. 
- `from dbt.adapters.contracts.connection import AdapterResponse, Credentials`

Reply:
@hanslemm, the suggested changes has been included. Please review the updated code. Thanks.